### DISCUSSION TITLE
Compatibility with scheduler PBSPRO and Centos8

### DISCUSSION THREAD

**Discussion by MahmoudAliAfifi** [nlmixr2/nlmixr2#72] [Q&A]
We want to run a massive number of models and we got access to a grid, and they are using Linux, and their grid is using PBSPRO scheduler.
Is nlmixr compatible with the PBSPRO scheduler?
Also is it compatible with Linux centos 8?

**Comment by mattfidler** [comment]
I have not used `PBSPRO` scheduled nor have I used `Linux Centos 8`, so I am unsure.

However, `nlmixr2` is tested on `fedora` which if I remember my linux geneolog correctly, is related to `Centos` so I believe it should work.  I also believe `PBSPRO` should work if it can schedule R.
